docs/schema-guide/tool.md

```diff
@@ -1 +1,12 @@
-Tool
+- A valid migration backend must export a value `migrationBackend :: BeamMigrationBackend be commandSyntax`
+- Use it by specifying the `--backend` along with the module i.e. `--backend Database.Beam.Postgres.Migration`
+
+- A schema module must export a value `migration :: MigrationSteps syntax () a'`
```

Comment:
Can we mark this work in progress? This is almost correct, but the `nextgen-beam-migrate` branch has several changes.